rewrite-java/src/main/java/org/openrewrite/java/cleanup/UseObjectNotifyAll.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.cleanup;
+
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.internal.lang.Nullable;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.MethodMatcher;
+import org.openrewrite.java.search.UsesMethod;
+import org.openrewrite.java.tree.J;
+
+import java.time.Duration;
+import java.util.Collections;
+import java.util.Set;
+
+public class UseObjectNotifyAll extends Recipe {
+
+    private static final MethodMatcher OBJECT_NOTIFY = new MethodMatcher("java.lang.Object notify()");
+
+    @Override
+    public String getDisplayName() {
+        return "Replaces `Object.notify()` with `Object.notifyAll()`";
+    }
+
+    @Override
+    public String getDescription() {
+        return "`Object.notifyAll()` and `Object.notify()` both wake up sleeping threads, but `Object.notifyAll()` only rouses one while`Object.notifyAll()` rouses all of them. " +
```

Comment:
```suggestion
        return "`Object.notifyAll()` and `Object.notify()` both wake up sleeping threads, but `Object.notify()` only rouses one while`Object.notifyAll()` rouses all of them. " +
```
Hope this is correct; it felt off reading it from the context.

Reply:
That was a typo that resulted from copying/pasting. Thank you for catching it :)